tf2_ros/include/tf2_ros/buffer.h

```diff
@@ -252,6 +252,14 @@ class Buffer : public BufferInterface, public AsyncBufferInterface, public tf2::
       callback);
   }
 
+  /**
+   * \brief Cancel the future to make sure the callback of requested transform is clean.
+   * \sa cancel(TransformStampedFuture& ts_future);
```

Comment:
Nit: In addition to the `\sa`, I think it is better to add a `\param` with documentation of the parameter.  It's a little redundant, but makes it easier on the reader of the API docs.